packages/runtime/package.json

```diff
@@ -22,5 +22,9 @@
     "react": "^18.2.0",
     "tsconfig": "workspace:*",
     "tsup": "^7.2.0"
+  },
+  "dependencies": {
+    "@mui/icons-material": "^5.14.9",
+    "@mui/material": "^5.14.9"
```

Comment:
Please move this to dev dependencies. Since this is a package, transitive dependencies are usually not bundled and added to peer dependencies so it's only need during dev time.